notebooks/models/system_linear_regression.ipynb

```diff
@@ -841,283 +1227,538 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
    ],
-   "metadata": {
-    "collapsed": false
-   }
+   "source": [
+    "from sklearn.preprocessing import MinMaxScaler\n",
+    "\n",
+    "data_columns = X.columns\n",
+    "scaler = MinMaxScaler()\n",
+    "X = pd.DataFrame(scaler.fit_transform(X))\n",
+    "X.columns = data_columns\n",
+    "X.head()"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": 53,
+   "execution_count": 47,
+   "metadata": {},
    "outputs": [],
    "source": [
     "from sklearn.model_selection import train_test_split\n",
     "\n",
-    "x_train, x_test, y_train, y_test = train_test_split(\n",
-    "    x, y, test_size=0.25, random_state=0, shuffle=True\n",
+    "X_train, X_test, y_train, y_test = train_test_split(\n",
+    "    X, y, test_size=0.25, random_state=0, shuffle=True\n",
     ")"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
     "## Regresja liniowa"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": 59,
+   "execution_count": 48,
+   "metadata": {},
    "outputs": [],
    "source": [
     "from sklearn.linear_model import LinearRegression\n",
     "\n",
     "\n",
     "model = LinearRegression()\n",
-    "model.fit(x_train, y_train)\n",
-    "predict = model.predict(x_test)"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+    "model.fit(X_train, y_train)\n",
+    "predict_train = model.predict(X_train)\n",
+    "predict = model.predict(X_test)"
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
     "## Sprawdzenie modelu"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": 60,
+   "execution_count": 49,
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "R2 score: 0.98\n",
-      "Mean square error: 0.59\n"
+      "R2 score:0.98\n"
      ]
     }
    ],
    "source": [
-    "from sklearn.metrics import mean_squared_error\n",
     "from sklearn.metrics import r2_score\n",
     "\n",
-    "\n",
-    "r2 = r2_score(y_test, predict)\n",
-    "MSE = mean_squared_error(y_test, predict)\n",
-    "print(\"R2 score:\", round(r2, 2))\n",
-    "print(\"Mean square error:\", round(MSE, 2))"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+    "predict_train = model.predict(X_train)\n",
+    "r2 = r2_score(y_train, predict_train)\n",
+    "print(f\"R2 score:{r2:.2f}\")"
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "* R2 - blisko 1, więc spoko\n",
-    "* trochę gorzej z mse byłoby fajnie jakby to była wartość bliżej 0, ale możliwe, że jak się zastosuje jakieś skalowanie\n",
-    "cech czy inną magię to będzie lepiej"
+    "R2 jest blisko 1, więc spoko"
```

Comment:
te moje okropne komentarze
ehhh
można to jakoś formalniej napisać, ale nie naciskam